### ISSUE TITLE
Improve the layout of the site 

### ISSUE THREAD

**Issue by koookee** [sonalij06/Know_Your_Lucky_Number.github.io#1]
It would look better if the layout was centered and symmetrical. The CSS could also be improved
![image](https://user-images.githubusercontent.com/55768904/193704956-cc1a2914-0ed2-464c-a443-ebc5a79d9e7c.png)

**Comment by ChancellorCeti** [comment]
I would like to work on the CSS, could you assign this issue to me?

**Comment by imdevarsh** [comment]
I would also like to work on this issue, can you please assign me? Thank you

**Comment by sonalij06** [comment]
> I would like to work on the CSS, could you assign this issue to me?

assigned

**Comment by InquisitiveMe15** [comment]
Hello! @sonalij06 

I just forked this repo after running it I got this.

![image](https://user-images.githubusercontent.com/85685489/193854800-77c8b663-0989-495f-9589-8b3d4a7f9cde.png)


Earlier it was...

![image](https://user-images.githubusercontent.com/85685489/193855244-74d49303-a9d1-4fd9-85c8-6d1c4d840f01.png)


Some words are not clear.

As this issue isn't closed, you can reassign this to me.

**Comment by koookee** [comment]
Sorry about that. It was an issue with the merge conflict. UI should be fine once the pull request is merged in

**Comment by InquisitiveMe15** [comment]
If I am not wrong the pull request is already merged.

**Comment by koookee** [comment]
Yes, the issue has been fixed. If you pull the changes it should update it to what it was originally supposed to look like.

**Comment by InquisitiveMe15** [comment]
I think you haven't created any other PR.  The one you have created is already merged in. If you have done changes to your local then create another PR for that.

**Comment by koookee** [comment]
I've created two pull requests so far (both got merged in). The first one improves the HTML layout of the page and the UI. The second one was just a quick fix to the merge conflict error that happened (which resulted in the page breaking). This issue, "Improve the layout of the site", is still open. If you want to take it on, feel free to do so. The html layout has already been fixed but we could still update the css of the page. 

If you pull in the latest changes, you should see the page with the fixed layout.

**Comment by InquisitiveMe15** [comment]
> I've created two pull requests so far (both got merged in). The first one improves the HTML layout of the page and the UI. The second one was just a quick fix to the merge conflict error that happened (which resulted in the page breaking). This issue, "Improve the layout of the site", is still open. If you want to take it on, feel free to do so. The html layout has already been fixed but we could still update the css of the page.
> 
> If you pull in the latest changes, you should see the page with the fixed layout.

ohkay fine

**Comment by NNeedlemouse** [comment]
> It would look better if the layout was centered and symmetrical. The CSS could also be improved ![image](https://user-images.githubusercontent.com/55768904/193704956-cc1a2914-0ed2-464c-a443-ebc5a79d9e7c.png)

I already made a fork and did a PR. Hope it looks a little bit better

**Comment by ghost** [comment]
I already did a fork and PR it into the repo. Hope it looks a bit better
